package.json

```diff
@@ -50,6 +50,7 @@
     "semantic-release": "semantic-release"
   },
   "dependencies": {
+    "@popperjs/core": "^2.11.8",
```

Comment:
@adamdehaven The reason why this was added:
https://github.com/nathanreyes/v-calendar/issues/853

Is there an alternative way to add the direct path to popper.js in the root `vite.config.ts`'s resolved aliases, so we can avoid adding this dependency?

Reply:
@adamdehaven is there any concerns to add this?

Reply:
Yes, possibly. 

We already have a dependency on the old popper.js package, so I'm unsure what effect(s) importing (and exporting/packaging) both will have. 

I can take a look when I'm back; since no repos should be picking up the alpha version yet I don't think there should be a rush to get this merged.

Reply:
Sounds good, i'll let it hang then.

Without it, `v-calendar` breaks on `yarn docs:build` with 

```
Rollup failed to resolve import "@popperjs/core
```

<img width="1174" alt="Screenshot 2023-09-20 at 2 21 41 PM" src="https://github.com/Kong/kongponents/assets/103061463/58ae14a1-89b2-436a-9538-51cd6edfacbe">

Reply:
It only breaks in the docs build, or if you pull in the preview Kongponents package and try to use it?

Reply:
It breaks on a `yarn build`

Reply:
I don't see any issues in the namespace with both dependencies (other than a larger package size)

@jillztom we could likely refactor our popover usages in this major version to get off of the old `"popper.js": "^1.16.1"` dependency as part of the `beta` work, before major release